src/main/java/com/sourcegraph/vcs/ConvertUtil.kt

```diff
@@ -1,55 +1,44 @@
 package com.sourcegraph.vcs
 
-import java.net.URL
-import java.util.regex.Pattern
+import java.net.URI
 
 fun convertGitCloneURLToCodebaseNameOrError(cloneURL: String): String {
```

Comment:
Note: does `codebaseName` deserve its own type? It's `String` now, it's easy to confuse it with the clone url string.

Reply:
I'm not doing this in this PR but generally I think it's good idea to change that when we will touch this code again.